### ISSUE TITLE
Unexepcted Token {: Importing Angular Lib into App

### ISSUE THREAD

**Issue by mousedownmike** [just-jeb/angular-builders#603]
**Describe the bug**
I've created two "simple" repositories to reproduce this but in summary I can't get past this:
```
/home/mike/git/jest-demo-app/node_modules/aws-amplify-angular/dist/src/providers/amplify.service.js:15
    import { Injectable, Optional, Inject } from '@angular/core';
           ^

    SyntaxError: Unexpected token {

      at ScriptTransformer._transformAndBuildScript (node_modules/@jest/transform/build/ScriptTransformer.js:537:17)
```
I am publishing an Angular library from `jest-demo-lib` and including it in `jest-demo-app`.  These are separate repositories.  Everything works fine until I import `aws-amplify-angular` in the library and attempt to use it in the app.

The commit histories of these repositories outline the step by step resolution of various issues described elsewhere but I can find no way of resolving this one.  Googling for "Unexpected token {" brings up the *other* unexpected token issues (i.e. `export` and `import`) that I have already resolved thanks to documentation found here and elsewhere.

**To Reproduce**
The App - This is where the error is occurring.
https://github.com/mousedownmike/jest-demo-app

1. Clone
2. npm install (this will pull the published `@mousedownco/jest-demo-lib`)
3. ng test
4. Observe `Unexpected token {`

The Library - This may contain useful context or maybe it's the problem.
https://github.com/mousedownmike/jest-demo-lib
[issues resolved by commit](https://github.com/mousedownmike/jest-demo-lib/commits/develop)
NPM published lib: `@mousedownco/jest-demo-lib`
https://www.npmjs.com/package/@mousedownco/jest-demo-lib

**Expected behavior**
The test should run.

**Builder:**

    "@angular-builders/jest": "^8.2.0",

**Libraries**

    "@angular-devkit/build-angular": "~0.803.8",

    "@aws-amplify/core": "^1.2.4",
    "aws-amplify-angular": "^3.1.4",

**Additional context**
I experienced this in a much larger application I'm trying to convert to Jest but I've started over from scratch with these two repos and the issue is now identical.  My understanding of the `transformIgnorePatterns` value may not be complete but [the values I've set](https://github.com/mousedownmike/jest-demo-app/blob/develop/jest.config.js#L7) have resolved the better documented `import` and `export` token issues.

I'm wondering if I need to change the way the `jest-demo-lib` is packaged so that is more usable by `jest-demo-app` and Jest but I'm not sure where to start looking.

Please let me know if this should go in another project.

Many thanks,
Mike

**Comment by mousedownmike** [comment]
This appears to be related to: https://github.com/facebook/jest/issues/8738

I'm going to see if I can update my demo apps to use a rolled back version.

**Comment by just-jeb** [comment]
Any updates?

**Comment by mousedownmike** [comment]
Nothing yet...  I'm following that babel-jest issue in case it gets resolved there.  The layers of abstraction here are more than I've been able to get my head around at the moment so I haven't dug into it too much deeper.  I'm leaving those repositories up until I figure this out.

**Comment by mousedownmike** [comment]
I've restructured my project to have a single Angular workspace with my libs and apps all in the same workspace.  I just did an update and this still seems to be an issue with the sample repos I put together (even when updated to the latest libs).  

I'm going to go ahead and close the issue as it will no longer be affecting me.  I'm leaving the sample repos up in case they're of any use to anyone.

Thanks again @just-jeb for this builder, your efforts are appreciated!
